mllib/src/main/scala/org/apache/spark/ml/r/SparkRWrappers.scala

```diff
@@ -52,6 +53,43 @@ private[r] object SparkRWrappers {
     pipeline.fit(df)
   }
 
+  def fitAFTSurvivalRegression(
+      value: String,
+      df: DataFrame): PipelineModel = {
+
+    def formulaRewrite(value: String): (String, String) = {
+      var rewrited: String = null
+      var censorCol: String = null
+
+      val regex = "^Surv\\s*\\(([^,]+),([^,]+)\\)\\s*\\~\\s*(.+)".r
```

Comment:
Use triple quotes to avoid escaping `\`. This should work: `"""Surv\((\S+), (\S+)\) ~ (\S+)""".r`.